src/EnergyPlus/DXCoils.cc

```diff
@@ -17346,7 +17346,7 @@ void ControlVRFIUCoil(EnergyPlusData &state,
     MaxSC = 20;
     Garate = state.dataDXCoils->DXCoil(CoilIndex).RatedAirMassFlowRate(1);
     // why always limit the minimum fan speed ratio to 0.65?
-    FanSpdRatioMin = min(max(OAMassFlow / Garate, 0.65), 1.0); // ensure that coil flow rate is higher than OA flow rate
+    FanSpdRatioMin = min(max(OAMassFlow / Garate, 0.0), 1.0); // ensure that coil flow rate is higher than OA flow rate
```

Comment:
This seems like a big change, unrelated to the heating rate outputs. Does this branch include some changes from another branch?

Reply:
I might have left it in by accident. Just reverted it.